src/components/events/Event/index.js

```diff
@@ -45,6 +46,24 @@ module.exports = React.createClass({
     };
   },
 
+  activate: function() {
+    this.setState({ activeClass: 'active' });
+  },
+
+  activateAndTimeout: function() {
+    this.activate();
+    this.timeout();
+  },
+
+  timeout: function() {
+    var _this = this;
```

Comment:
I don't particular mind. But why do you think "component" is more obvious? It implies you should never use the "this" object anywhere, instead always making a "component" variable to call everything on.